### PR TITLE
Implements new ERT system.

### PR DESCRIPTION
**Pull request by Andromeda-K22**
50% chance of frontier hicks
20% of a random, helpful vessel named the ISV Helping Hand
10% chance of pirates
10% chance of solgov enforcers
10% chance of a syndicate strike team.

### CODE REVIEW

**Comment by Andromeda-K22** [comment]
will fix this whenever the fuck i figure out how to fix the conflicts.

File: code/game/antagonist/outsider/distress_responder.dm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+/datum/antagonist/distress_response
+	id = MODE_DISTRESS
+	role_text = "Distress Responder"
+	role_text_plural = "Distress Responders"
+	welcome_text = "As member of the Distress Responders, you answer to your leader and the admins."
+	antag_text = "You are a potential antagonist! Pay attention to the stated text and your mission."
+	leader_welcome_text = "You shouldn't see this"
+	landmark_id = "Response Team"
+	id_type = /obj/item/weapon/card/id/centcom/ERT
```
Comment (MalfunctioningAI) [inline code/game/antagonist/outsider/distress_responder.dm]: Why would a random vessel have access to the doors inside of another random vessel?

File: code/game/antagonist/outsider/distress_responder.dm [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	var/antag_objective = null //We're special so this gets handled here!
+
+	flags = ANTAG_OVERRIDE_JOB | ANTAG_SET_APPEARANCE | ANTAG_HAS_LEADER | ANTAG_CHOOSE_NAME | ANTAG_RANDOM_EXCEPTED
+	antaghud_indicator = "hudloyalist"
+
+	hard_cap = 5
+	hard_cap_round = 7
+	initial_spawn_req = 5
```
Comment (MalfunctioningAI) [inline code/game/antagonist/outsider/distress_responder.dm]: 5 people sounds a bit too much. Maybe 3-4?

File: code/game/distress_call.dm [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+/datum/game_mode/proc/setup_equipt() //Set up our equiptment tags.
+	var/datum/distress_call/P
+	P = picked_call
+	distress_team_type = P.equipt_tag
+
+/datum/game_mode/proc/show_join_message()
+	for(var/mob/M in world)
+		if(isghost(usr) || isnewplayer(usr))
+			if(M.client)
+				to_chat(M, "<font size='3'>\red An emergency beacon has been activated. Use the <B>Join Distress Team</b> verb, <B>IC tab</b>, to join!</font>")
```
Comment (MalfunctioningAI) [inline code/game/distress_call.dm]: \<span class='danger'>TEXTHERE\</span>
^ Red text.

File: code/game/distress_call.dm [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+	can_call_ert = 0 // Only one call per round, gentleman.
+	send_distress_team = 1
+	distress_equiptment_spawn()
+	show_join_message()
+	spawn(600 * 5)
+		send_distress_team = 0 // Can no longer join the ERT.
+
+//Variable equiptment spawning handled below
+
+/datum/game_mode/proc/distress_equiptment_spawn()
```
Comment (MalfunctioningAI) [inline code/game/distress_call.dm]: equipment*

**Comment by MalfunctioningAI** [comment]
Squash all of your commits for the sake of tidyness, if you can.

**Comment by Andromeda-K22** [comment]
No reviewers available. Travis runs fine. Merging.

File: code/game/antagonist/outsider/distress_responder.dm (hunk 1 above)
Comment (Andromeda-K22) [reply code/game/antagonist/outsider/distress_responder.dm]: This is for the sake of simplicity. I would rather not deal with them not even being able to get through the airlock.

File: code/game/distress_call.dm (hunk 3 above)
Comment (Andromeda-K22) [reply code/game/distress_call.dm]: This is intentional.

**Comment by Andromeda-K22** [comment]
fam i don't know how.

File: code/game/distress_call.dm (hunk 4 above)
Comment (Andromeda-K22) [reply code/game/distress_call.dm]: Why are we caring about spelling in proc names?

File: code/game/antagonist/outsider/distress_responder.dm (hunk 1 above)
Comment (MalfunctioningAI) [reply code/game/antagonist/outsider/distress_responder.dm]: Doors near airlocks are common use, and the airlocks are opened by the airlock controllers. They still need docking codes to open the airlocks, but an ID is never required to open an airlock.

**Comment by MalfunctioningAI** [comment]
Don't worry.

File: code/game/antagonist/outsider/distress_responder.dm (hunk 1 above)
Comment (MalfunctioningAI) [reply code/game/antagonist/outsider/distress_responder.dm]: @Andromeda-K22

**Pull request merged**